cmd/utils/flags.go

```diff
@@ -389,6 +389,11 @@ var (
 		Usage: "The raft ID to assume when joining an pre-existing cluster",
 		Value: 0,
 	}
+	RaftPortFlag = cli.IntFlag{
```

Comment:
Maybe `UintFlag` would be better

Reply:
I was thinking the same, but decided at the time to follow suit with other existing port flags.